### ISSUE TITLE
Other network streaming protocols?

### ISSUE THREAD

**Issue by kamil-bednarek** [cedricfung/MediaCodecDemo#3]
Is there any way to use that method for example for RTMP based streaming?

**Comment by cedricfung** [comment]
Yes, I recommend you to watch https://github.com/google/ExoPlayer, it has better documentation to add support for new different media formats or network protocols.
